tests/UI/campaigns/functional/API/01_clientCredentialGrantFlow/01_internalAuthServer/02_resourceEndpoint.ts

```diff
@@ -122,62 +119,6 @@ describe('API : Internal Auth Server - Resource Endpoint', async () => {
       const jsonResponse = await apiResponse.json();
       expect(jsonResponse).to.have.property('access_token');
       expect(jsonResponse.access_token).to.be.a('string');
-
-      accessToken = jsonResponse.access_token;
-      accessTokenExpired = utilsAPI.setAccessTokenAsExpired(accessToken);
-    });
-
-    it('should request the endpoint /hook-status/1 without access token', async function () {
-      await testContext.addContextItem(this, 'testIdentifier', 'requestEndpointWithoutAccessToken', baseContext);
-
-      const apiResponse = await apiContext.get('hook-status/1');
```

Comment:
Please replace hook-status by hooks

(We don't test the endpoint but the access)